frontend/app/styles/main.less

```diff
@@ -83,6 +83,18 @@ body {
   }
 }
 
+.login-form {
+  margin: 5% 34%;
+  width: 30%;
+  background-color: #FFFFFF;
+  color: #000;
+  padding: 5% 2% 5% 2%;
```

Comment:
Olalala! There are percents as padding! I cried, when I look this code! Please do not do it so ever!!